tasks/build-fits-site.yml

```diff
@@ -0,0 +1,83 @@
+- name: Build site for FITS microservice
+  blockinfile:
+    path: "/etc/apache2/conf-available/CrayFits.conf"
```

Comment:
Should use `{{ httpd_conf_directory }}` to allow for changing Apache install directory.